### PR TITLE
Add az for aws on machine config step cluster creation

### PR DESCRIPTION
**Pull request by nikage**

### CODE REVIEW

File: cmd/ui/assets/src/app/shared/supergiant/cloud-accounts/cloud-accounts.service.ts [hunk 1]
```diff
@@ -22,7 +23,7 @@ export class CloudAccounts {
     return this.util.fetch(this.cloudAccountsPath + '/' + cloudAccountName + '/' + 'regions' + '/' + region + '/az');
   }
 
-  public getAwsMachineTypes(cloudAccountName, region, az) {
+  public getAwsMachineTypes(cloudAccountName, region, az):Observable<string[]> {
```
Comment (josephkane) [inline cmd/ui/assets/src/app/shared/supergiant/cloud-accounts/cloud-accounts.service.ts]: did this break new cluster provisioning?

**Comment by gopherstein** [comment]
Reference the issue this fixes in the pull.

File: cmd/ui/assets/src/app/shared/supergiant/cloud-accounts/cloud-accounts.service.ts (hunk 1 above)
Comment (nikage) [reply cmd/ui/assets/src/app/shared/supergiant/cloud-accounts/cloud-accounts.service.ts]: no. Why?

Comment (josephkane) [reply cmd/ui/assets/src/app/shared/supergiant/cloud-accounts/cloud-accounts.service.ts]: nope, your right. everythings good, my bad

**Pull request merged**